quinn-udp/src/lib.rs

```diff
@@ -132,7 +132,7 @@ impl Default for RecvMeta {
 }
 
 /// An outgoing packet
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I think it would make sense to squash into the previous (`BytesMut` -> `Bytes`) commit as well.